www/proxy-sso/Makefile

```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=		proxy-sso
```

Comment:
are you ok with replacing the old plugin? the name would be web-proxy-sso or web-proxy-sso2, although that gets confusing...

Reply:
no problen, as you wish )